### ISSUE TITLE
Problem in getting Correct List Of Search Results

### ISSUE THREAD

**Issue by SomaiyehTaheri** [Shazwazza/Examine#286]
Hi 

We are facing the same problem with examine search all fields in two different websites which are in Umbraco 8.14.0
when searching for a particular word one of the Contents that contain the exact search term does not appear in the returned list 

![image](https://user-images.githubusercontent.com/52094360/177211394-bb5261e4-0355-495a-8e37-9a8a35ec1cbb.png)

this page which have fongogo word in website field dose not returned as a search result 
![image](https://user-images.githubusercontent.com/52094360/177211731-881631e5-1ed7-46d1-a4a6-2150f900645a.png)

**Comment by Shazwazza** [comment]
If you rebuild your index does it show up?

**Comment by SomaiyehTaheri** [comment]
> If you rebuild your index does it show up?

no, it does not, I rebuild the index a few times, I also clean the temp folders but had no success

**Comment by Shazwazza** [comment]
So the problem potentially is it's indexing a URL with that term in it and it may depend on the analyzer being used. Are you searching the internal or external index? Does it show up in the external index? The internal index uses a whitespace analyzer, the external one uses a standard analyzer. The standard analyzer will tokenize that URL but the whitespace one will not.

**Comment by SomaiyehTaheri** [comment]
I checked in both of them(internal and external), but it does not show up in any of them

**Comment by Shazwazza** [comment]
You can check your indexes with Luke https://code.google.com/archive/p/luke/downloads for lucene 3.x to see how that field is indexed for that document. 

You can also search on that document Id in the examine dashboard to make sure that value exists in that document.

**Comment by SomaiyehTaheri** [comment]
> You can check your indexes with Luke https://code.google.com/archive/p/luke/downloads for lucene 3.x to see how that field is indexed for that document.
> 
> You can also search on that document Id in the examine dashboard to make sure that value exists in that document.

thanks for your help, I will check.
